BDN does not build using dotnet sdk from the command line in Linux

When checking out master copy of BDN in Linux and issuing `dotnet build` the build fails due to missing reference assemblies on Linux.

```bash
...
XUnit/TheoryWindowsOnlyAttribute.cs(9,43): error CS0518: Predefined type 'System.String' is not defined or imported [/home/dmg/projects/public/BenchmarkDotNet/
tests/BenchmarkDotNet.Tests/BenchmarkDotNet.Tests.csproj]
XUnit/TheoryWindowsOnlyAttribute.cs(9,9): error CS0518: Predefined type 'System.Void' is not defined or imported [/home/dmg/projects/public/BenchmarkDotNet/tes
ts/BenchmarkDotNet.Tests/BenchmarkDotNet.Tests.csproj]
    155 Warning(s)
    11220 Error(s)

Time Elapsed 00:00:13.92
```

Full build log:
https://gist.github.com/damageboy/09c715f9159f36d92813f31323d4b981

--- a/build/common.props
+++ b/build/common.props
@@ -57,9 +57,10 @@
     <InformationalVersion>$(Major).$(Minor).$(Revision).$(BuildNumber)$(PrereleaseLabel)</InformationalVersion>
     <PackageVersion>$(Major).$(Minor).$(Revision).$(BuildNumber)$(PrereleaseLabel)</PackageVersion>
   </PropertyGroup>
-  <PropertyGroup>
-    <TargetFrameworkRootPath Condition="'$(TargetFrameworkRootPath)' == '' AND Exists('/Library/Frameworks/Mono.framework/Versions/Current')">/Library/Frameworks/Mono.framework/Versions/Current/lib/mono/xbuild-frameworks</TargetFrameworkRootPath>
-    <TargetFrameworkRootPath Condition="'$(TargetFrameworkRootPath)' == '' AND Exists('/usr/local/lib/mono/xbuild-frameworks')">/usr/local/lib/mono/xbuild-frameworks</TargetFrameworkRootPath>
-    <TargetFrameworkRootPath Condition="'$(TargetFrameworkRootPath)' == '' AND Exists('/usr/lib/mono/xbuild-frameworks')">/usr/lib/mono/xbuild-frameworks</TargetFrameworkRootPath>
-  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
 </Project>

--- a/samples/BenchmarkDotNet.Samples.FSharp/BenchmarkDotNet.Samples.FSharp.fsproj
+++ b/samples/BenchmarkDotNet.Samples.FSharp/BenchmarkDotNet.Samples.FSharp.fsproj
@@ -19,6 +19,10 @@
   </ItemGroup>
   
   <ItemGroup>
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
     <PackageReference Update="FSharp.Core" Version="4.6.0" />
     <PackageReference Update="System.ValueTuple" Version="4.5.0" />
   </ItemGroup>

--- a/src/BenchmarkDotNet.Disassembler.x64/BenchmarkDotNet.Disassembler.x64.csproj
+++ b/src/BenchmarkDotNet.Disassembler.x64/BenchmarkDotNet.Disassembler.x64.csproj
@@ -1,4 +1,5 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
   <PropertyGroup>
     <TargetFramework>net461</TargetFramework>
     <OutputType>Exe</OutputType>

--- a/src/BenchmarkDotNet.Disassembler.x86/BenchmarkDotNet.Disassembler.x86.csproj
+++ b/src/BenchmarkDotNet.Disassembler.x86/BenchmarkDotNet.Disassembler.x86.csproj
@@ -1,4 +1,5 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />      
   <PropertyGroup>
     <TargetFramework>net461</TargetFramework>
     <OutputType>Exe</OutputType>

